### PR TITLE
chore: Price DB copy update

### PR DESCRIPTION
**Pull request by rajsam003**
This PR resolves [CX-2346](https://artsyproduct.atlassian.net/browse/CX-2346) 

|Before|
|---|
|image|
|After|
|image|

### CODE REVIEW

File: src/v2/Apps/PriceDatabase/Components/PriceDatabaseBenefits.tsx [hunk 1]
```diff
@@ -19,24 +19,34 @@ export const PriceDatabaseBenefits: React.FC = () => {
...
 
       <Section
-        title="The largest publicly available art market database"
-        text="With millions of auction records, our growing database encompasses over 900 auction houses with pricing that dates back to 1986. Get real-time records from history-making auction houses like Christie's, Sotheby's, Phillips, Bonhams, and more."
+        title="Get in-depth art market data"
+        description={
+          <Text variant="sm">
+            Browse millions of current and historical results from leading
+            auction houses across the globe.
+          </Text>
```
Comment (olerichter00) [inline src/v2/Apps/PriceDatabase/Components/PriceDatabaseBenefits.tsx:35]: What is the reason for this change?

**Review by olerichter00 — COMMENTED** [review]
Thanks for updating the copy!

Will there be a follow-up to add a section for Value Prop 4?

File: src/v2/Apps/PriceDatabase/Components/PriceDatabaseBenefits.tsx (hunk 1 above)
Comment (rajsam003) [reply src/v2/Apps/PriceDatabase/Components/PriceDatabaseBenefits.tsx:35]: @MounirDhahri 👀?

**Review by olerichter00 — APPROVED** [review]

File: src/v2/Apps/PriceDatabase/Components/PriceDatabaseBenefits.tsx (hunk 1 above)
Comment (MounirDhahri) [reply src/v2/Apps/PriceDatabase/Components/PriceDatabaseBenefits.tsx:35]: we just spoke of this during the standup.

Comment (MounirDhahri) [reply src/v2/Apps/PriceDatabase/Components/PriceDatabaseBenefits.tsx:35]: @olerichter00 yes, we will add the value prop 4 once we get the image for value prop 4 from the Brand team

**Review by MounirDhahri — APPROVED** [review]

File: src/v2/Apps/PriceDatabase/Components/PriceDatabaseBenefits.tsx [hunk 2]
```diff
@@ -70,10 +86,10 @@ export const PriceDatabaseBenefits: React.FC = () => {
 
 const Section: React.FC<{
   title: string
-  text: string
+  description: ReactElement
```
Comment (MounirDhahri) [inline src/v2/Apps/PriceDatabase/Components/PriceDatabaseBenefits.tsx:89]: can we revert to using text as a prop here until we get the value prop 4

**Review by MounirDhahri — CHANGES_REQUESTED** [review]

File: src/v2/Apps/PriceDatabase/Components/PriceDatabaseBenefits.tsx (hunk 1 above)
Comment (MounirDhahri) [reply src/v2/Apps/PriceDatabase/Components/PriceDatabaseBenefits.tsx:35]: But I agree, it makes sense to revert this until we need it
